Automattic/jetpack
# Blaze: enable new dashboard by default.

## Proposed changes:

This should be the final step to enable the new Blaze dashboard to all self-hosted site owners.

### Other information:


## Jetpack product discussion

* p1688742916485729/1688741150.234289-slack-C053FAERCDP
* #30103
* p1689793730589839-slack-C053FAERCDP

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a self-hosted site that's connected to WordPress.com.
* You should now see a Tools > Advertising menu in wp-admin.
* When clicking on it, you should be directed to the Blaze dashboard.
